### ISSUE TITLE
Password is wrong

### ISSUE THREAD

**Issue by fungusrocks** [Berry-Pool/nami-wallet#227]
Hey I got "Password is wrong" when I was trying to send ada(I believe it's correct since I use password manger) and just got stuck there. Unable to retrieve pwd. 

And somehow got the same error message when I tried to create a new account to reimport my yoroi wallet. Any thoughts?
<img width="318" alt="Screen Shot 2022-01-17 at 10 04 58 PM" src="https://user-images.githubusercontent.com/97936399/149880055-53f3f0cf-2982-496c-a51e-22244e1bd7cb.png">

**Comment by Cjunk** [comment]
Yes I am getting password is wrong too however I always use the same password. I am here trying to workout where NAMI stores the password (hashed no doubt).
I know my passwords are correct as no where else has the issue.
